marketplace/src/test/java/com/cosmocats/marketplace/external/PricingClientWiremockTest.java

```diff
@@ -0,0 +1,120 @@
+package com.cosmocats.marketplace.external;
+
+import com.cosmocats.marketplace.external.dto.ExternalPriceDTO;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+// ⬇️ 1. ДОДАНО НОВИЙ ІМПОРТ
+import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+@SpringBootTest
+@DisabledIfEnvironmentVariable(
```

Comment:
what?

Reply:
You have wiremock configured wrongly. Basically you are managing one wire mock instance per test class, this approach is to heavy and will work purly for the projects with more tests